### DISCUSSION TITLE
Hospital Price Transparency

### DISCUSSION THREAD

**Discussion by joryhatton** [CMSgov/price-transparency-guide#468] [Q&A] [answered]
Will a separate repository be created to further develop and establish similar conventions for the Hospital Price Transparency machine-readable file requirements?

**Comment by JR-CMS** [comment, marked as answer]
Please submit questions related to the Hospital Price Transparency requirements to our colleagues at PriceTransparencyHospitalCharges@cms.hhs.gov.

Thank you
